spec/controllers/account_activations_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AccountActivationsController, type: :controller do
+
```

Comment:
Do add some specs here eventually. There are no controller specs in the application yet, so I will add some soon (probably next friday) to give you an example.